components/Layout/Nav.tsx

```diff
@@ -132,37 +147,54 @@ const Nav = () => {
-            </div>
-            <div className={'animate-marquee2 absolute top-0 flex items-center whitespace-nowrap'}>
-              <div className={'rounded-full w-2 h-2 mx-2 bg-red-700 animate-pulse float-left'} />
-              <p className={'text-[#B999FA] text-[16px]'}> MINE.FM Live Broadcasts </p>
-            </div>
+      {isMobile ? (
+        <div
+          className={
+            'flex w-fit h-12 border-solid border-[#654A8F] rounded-md overflow-x-hidden overflow-y-hidden relative cursor-pointer md:ml-12'
+          }
+        >
+          <div
+            className={
+              'animate-marquee flex items-center justify-center pt-1 whitespace-nowrap'
+            }
+          >
+            <div
+              className={'rounded-full w-2 h-2 mx-2 bg-red-700 animate-pulse float-left'}
+            />
+            <p className={'text-[#B999FA] text-[16px]'}> MINE.FM Live Broadcasts </p>
           </div>
-        ) : (
-          <div className={'flex items-center justify-center w-2/5 h-12 border-solid border-[#654A8F] rounded-md overflow-x-hidden overflow-y-hidden relative md:left-12 lg:left-28 cursor-pointer mx-auto'}>
-            <div className={'flex items-center justify-center pt-1 whitespace-nowrap'}>
-              <div className={'rounded-full w-2 h-2 mx-2 bg-red-700 animate-pulse float-left'} />
-              <p className={'text-[#B999FA] text-center text-[16px]'}> MINE.FM Live Broadcasts </p>
-            </div>
-
+          <div
+            className={
+              'animate-marquee2 absolute top-0 flex items-center whitespace-nowrap'
+            }
+          >
+            <div
+              className={'rounded-full w-2 h-2 mx-2 bg-red-700 animate-pulse float-left'}
+            />
+            <p className={'text-[#B999FA] text-[16px]'}> MINE.FM Live Broadcasts </p>
           </div>
-        )
-      }
-
-      <div className={navActions}>
-        <Link
-          key={'about-us'}
-          href={'/about-us'}
+        </div>
+      ) : (
+        <div
+          className={
+            'flex items-center justify-center w-2/5 h-12 border-solid border-[#654A8F] rounded-md overflow-x-hidden overflow-y-hidden relative md:left-12 lg:left-28 cursor-pointer mx-auto'
+          }
         >
-          <p className={`m-2 text-xl text-[#B999FA] cursor-pointer hover:text-purple-800 ${isMobile ? 'hidden' : null}`}> ABOUT US </p>
+          <div className={'flex items-center justify-center pt-1 whitespace-nowrap'}>
+            <div
+              className={'rounded-full w-2 h-2 mx-2 bg-red-700 animate-pulse float-left'}
+            />
+            <p className={'text-[#B999FA] text-center text-[16px]'}>
+              {' '}
+              MINE.FM Live Broadcasts{' '}
+            </p>
+          </div>
+        </div>
+      )}
 
-        </Link>        <ConnectButton />
+      <div className={navActions}>
+        <ConnectButton />
```

Comment:
we should fix the spacing on the left here on desktop and mobile. We can just copy the spacing on the left side of the page where the minefm logo is so it doesn't look as awkward.

<img width="410" alt="Screenshot 2023-12-20 at 12 58 31 PM" src="https://github.com/Brick-Creative-Studio/mine-fm/assets/32021205/cec0e069-8936-43ec-97ca-3843328b7ae6">

Reply:
could you clarify what spacing you're talking about? Is it the left side of the connect wallet button or the right side of the page? And should the logo be that close to the Live Broadcasts crawler?

Reply:
lets get rid of the extra spacing on the right side of the page so that the menu and connect button are closer to the right edge of the page. And nah the live broadcast bar should have a bit more margin space on both sides.